src/PrestaShopBundle/Form/Admin/Sell/Product/BasicSettingsTabType.php

```diff
@@ -0,0 +1,86 @@
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product;
+
+use PrestaShopBundle\Form\Admin\Type\TranslatableType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class BasicSettingsTabType extends AbstractType
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('short_description', TranslatableType::class, [
+                'type' => TextareaType::class,
+                'options' => [
+                    'attr' => [
+                        'placeholder' => $this->translator->trans('The summary is a short sentence describing your product.<br />It will appears at the top of your shop\'s product page, in product lists, and in search engines\' results page (so it\'s important for SEO). To give more details about your product, use the "Description" tab.', [], 'Admin.Catalog.Help'),
+                    ],
+                ],
+            ])
+            ->add('description', TranslatableType::class, [
+                'type' => TextareaType::class,
+            ])
+            ->add('with_combinations', ChoiceType::class, [
+                'expanded' => true,
+                'multiple' => false,
+                'choices' => [
+                    $this->translator->trans('Simple product', [], 'Admin.Catalog.Feature') => 0,
+                    $this->translator->trans('Product with combinations', [], 'Admin.Catalog.Feature') => 1,
```

Comment:
Christophe suggested calling it _Product with attributes_ rather than _Product with combinations_ so that users can understand that combinations are based on attributes. I think it is a good idea.

Reply:
Hmm, I don't know, user combines attributes into combinations. What is more, "Combinations" word is used a lot in product page. There is even dedicated "Combinations" tab. :confused:

Reply:
Actually, they don't and this is problematic... it is not clear enough to users that attributes are made to create combinations. And since the "Combinations" tab is just down the wording, I think it won't be too confusing.

@marionf, @colinegin, can I ask your opinion about it?
😉

Reply:
@LouiseBonnard for me it's clear that combinations are created from attributes, see https://prnt.sc/ovj5rk

Reply:
@LouiseBonnard ok for me to change for _Product with attributes_

Reply:
```suggestion
                    $this->translator->trans('Product with attributes', [], 'Admin.Catalog.Feature') => 1,
```